applications/crossbar/src/modules_v2/cb_phone_numbers_v2.erl

```diff
@@ -940,14 +867,23 @@ identify(Context, Number) ->
 %%--------------------------------------------------------------------
 -spec read(cb_context:context(), ne_binary()) -> cb_context:context().
 read(Context, Number) ->
-    Options = [{'auth_by', cb_context:auth_account_id(Context)}
-               ,{'dry_run', not cb_context:accepting_charges(Context)}
-              ],
-    Result = case knm_number:get(Number, Options) of
-                 {'ok', KNum} -> {'ok', knm_number:to_public_json(KNum)};
-                 {'error', _R}=Error -> Error
-             end,
-    set_response(Result, Number, Context).
+    ViewOptions = [ {'keys', [knm_converters:normalize(Number)]}
+                  ],
+    case
+        cb_context:resp_data(
+          crossbar_doc:load_view(?CB_LIST, ViewOptions, Context, fun normalize_view_results/2)
```

Comment:
@jamesaimonetti this won't `reconcile` number (as `knm_number:get/2` would), is this ok you think?